StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement+Internal.swift

```diff
@@ -37,6 +37,18 @@ extension EmbeddedPaymentElement {
             isFlatCheckmarkStyle: configuration.appearance.embeddedPaymentElement.row.style == .flatWithCheckmark
         )
         let initialSelection: EmbeddedPaymentMethodsView.Selection? = {
+            // read from back end
+            if configuration.allowsSetAsDefaultPM,
+               let customer =  loadResult.elementsSession.customer {
+                let defaultPaymentMethod = customer.paymentMethods.filter {
+                    $0.stripeId == customer.defaultPaymentMethod
+                }.first
+                if let defaultPaymentMethod = defaultPaymentMethod {
+                    return .saved(paymentMethod: defaultPaymentMethod)
+                }
+                
+            }
+
```

Comment:
Somewhat similar comment to above, about the "read from back end" comment and the nested if's

Maybe:
```swift
if configuration.allowsSetAsDefaultPM,
   let customer = loadResult.elementsSession.customer,
   let defaultPaymentMethodId = customer.defaultPaymentMethod,
   let defaultPaymentMethod = customer.paymentMethods.first(where: { $0.stripeId == defaultPaymentMethodId }) {
    return .saved(paymentMethod: defaultPaymentMethod)
}
```